### ISSUE TITLE
Irritable Bowel Syndrome

### ISSUE THREAD

**Issue by LM-Ontologies** [obophenotype/human-phenotype-ontology#6486]
**Preferred term label:**
Irritable Bowel Syndrome

**Synonyms**
GI irritability

**Definition (free text, please give PubMed ID)**
Gastrointestinal disorder characterized by chronic abdominal pain and altered bowel habits in the absence of any organic cause. No abnormality is found on examination/procedures.

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0011024

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Irritbale bowel syndrome is a disease with multiple signs and symptoms. We would use the MONDO term for it : https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0005052

I wonder if we should make a phenotype term for "GI irritability" (which is one of the features of that disease, including also cramping, abdominal pain, bloating, gas, and diarrhea or constipation, or both.

I am not sure if there is a case to be made for this new term in addition to the other features, which already exist as HPO terms. Let me know if you think that irritability is a separate, observable phenotype (i.e., not the inference that the bowel must be "irritable"), and provide a definition -- otherwise, I think the existing terms are probably suffficient.

**Comment by pnrobinson** [comment]
Having a look at some reference material, I think the existing HPO terms are sufficient to cover the phenotypic abnormalities, and we should code "Irritable bowel syndrome" as a disease using the MONDO term mentioned above.
